Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -61,8 +62,8 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
 
     /// Returns the output obtained from applying the layer to the given input.
     ///
-    /// - Parameter input: The input to the layer `[batchCount, width, inputChannels]`.
-    /// - Returns: The output `[batchCount, newWidth, outputChannels]`.
+    /// - Parameter input: The input to the layer `[batch count, width, input channel count]`.
```

Comment:
You can call them "input width" and "output width" to be consistent with the channel counts.

Reply:
Since you're at it, would you mind adding an equation showing how the output width is computed? That is often helpful to people and it would probably be good to do for the 2D and 3D versions also.

Reply:
Sure thing! Good idea.

Reply:
Great, thanks! :)